src/views/ShipToStoreOrders.vue

```diff
@@ -309,15 +309,16 @@ export default defineComponent({
         };
         resp = await OrderService.shipOrder(payload);
         if (!hasError(resp)) {
-          // Send pickup notification email
-          resp = await OrderService.sendPickupScheduledNotification({ shipmentId });
-          // Refresh the incoming orders list
-          this.getIncomingOrders();
-
-          if (!hasError(resp)) {
-            showToast(translate('Order marked as ready for pickup, an email notification has been sent to the customer'));
-          } else {
+          try{
+            const emailResp = await OrderService.sendPickupScheduledNotification({ shipmentId });
+            if(!hasError(emailResp)){
+              showToast(translate('Order marked as ready for pickup, an email notification has been sent to the customer'));
+            }
+          }catch(error){
+            logger.error('Error sending pickup scheduled notification:', error);
```

Comment:
I think we do not need a separate try catch block to just handle the case of email error.